### PR TITLE
Use ontology terms where applicable (#31)

### PR DESCRIPTION
**Pull request by danielsotirhos**

### CODE REVIEW

**Comment by danielsotirhos** [comment]
A test case is needed as none of the current test cases contain a "ontology_label" field. Many have "text" and "ontology" fields but not "ontology_label".

File: src/humancellatlas/data/metadata/api.py [hunk 1]
```diff
@@ -721,3 +721,12 @@ def sequencing_output(self) -> List[SequenceFile]:
 }
 
 assert len(entity_types) == len(schema_names), "The mapping from schema name to entity type is not bijective"
+
+def ontology_label(d, default=RAISE):
+    """
+    Return a value from the specified dictionary using the first key found from a prioritized list of keys
```
[the remark this answers is not in the document]
Comment (hannes-ucsc) [reply src/humancellatlas/data/metadata/api.py]: I'll tweaked this in a subsequent commit.

**Comment by hannes-ucsc** [comment]
I squashed your two commits and added another one on top. Please review.

**Comment by hannes-ucsc** [comment]
Somehow I can't request a review from you since you're the author. I would have to PR against your branch. Next time I'll do that.

**Pull request merged**